src/metatensor/models/experimental/pet/train.py

```diff
@@ -16,6 +16,7 @@
 
 
 logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
```

Comment:
I don't think we are supposed to set the log level in each individual modules. Shouldn't this happen at the top level?

Reply:
It doesn't work otherwise. I tried debugging a bit and it looks like
`logger = logging.getLogger(__name__)`
is resetting it every time it's called

Reply:
what's the issue that `logger.setLevel(logging.INFO)` is trying to solve? Do you see something weird in the output?

Reply:
Yes, all the info-level messages weren't being printed. Using it only in the top modules (or in the hydra .yaml files) didn't fix it

Reply:
This works fine for me on a new install of metatensor-models. I remove this line from the SOAP-BPNN train script and still got 

```
[2024-03-11 12:12:17][INFO] - Calling architecture trainer
[2024-03-11 12:12:17][INFO] - Checking datasets for consistency
[2024-03-11 12:12:17][INFO] - Training on device cpu
```

Where the last two messages come from the train script

Reply:
It was, but somehow after this update it wasn't printing it anymore.
I'll try to see if I can fix it